docs/1_Core/Probability_&_Statistics/10_Comparing_Samples.md

```diff
@@ -0,0 +1,112 @@
+# Comparing Samples
+
+## $t$ Distribution DOF
+
+$$
+\text{DOF}
+=
+\dfrac{
+\Big[ \sum_{i=1}^2 (s_i^2/n_i) \Big]^2
+}{
+\sum_{i=1}^2 \dfrac{(s_i/n_i)^2}{n_i - 1}
+}
+$$
+
+## Comparing Means
+
+Using central-limit theorem, sampling distribution’s mean is normally-distributed, else t-distributed
+
+### $\hat \sigma_1 \ne \hat \sigma_2$
+
+Given
+
+- $(\bar x_1, s_1)$
+- $(\bar x_2, s_2)$
+
+$$
+\begin{aligned}
+z \text{ or } t
+&= \dfrac{(\bar x_1 - \bar x_2) - E[(\bar x_1 - \bar x_2)]}{\sigma^2(\bar x_1 - \bar x_2)}
+\\
+&= \dfrac{
+	(\bar x_1 - \bar x_2) - (\hat \mu_1 - \hat \mu_2)
+}{
+\sqrt{
+	\sum_{i=1}^2 \dfrac{s_i^2}{n_i} + 2 \rho_{12} s_1 s_2
+}
+}
+\end{aligned}
+$$
+
+Simplification: Is $\mu_1$ and $\mu_2$ statistically different? $\implies (\hat \mu_1 - \hat \mu_2)=0$
+
+### $\hat \sigma_1 = \hat \sigma_2$
+
+Pooled samples: If we are confident that the population variance are same, we can pool all data to make one estimate of the population variance
+
+$$
+\begin{aligned}
+s^2_{12}
+&= \dfrac{
+(n_1-1) s^2_1 + (n_2-1) s^2_2
+}{
+(n_1-1) + (n_2-1)
+}
+\end{aligned}
+$$
+
+### Pairing
+
```

Comment:
Selected a random page and hope this is just some weird browser thing on my side but this equation doesn't seem to render for me in the preview.

![image](https://github.com/user-attachments/assets/8bf4baa9-8ef9-47ff-8f48-21b9983afcfd)

Reply:
Happens when there is no empty line between text and $

![image](https://github.com/user-attachments/assets/fbca0162-6cd6-4761-867c-401ec05d6fb2)

I don't realize this because most markdown editors can still render correctly.

Reply:
This happens in a lot of places. Any idea of how to bulk fix this?

It's not as simple as adding a new line before every `$$`

This will break the ending `$$`

Reply:
Not sure, haven't had much experience with katex